### PR TITLE
Deprecate set-output

### PR DESCRIPTION
**Pull request by noatamir**
I noticed deprecation warning on `set-out` and followed the documentaiton here: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

I'm not sure experienced in solving such issues, so I wasn't sure how to test it, and suggestions on how to proceed would be welcome.

### CODE REVIEW

**Comment by oraNod** [comment]
@rossjrw Hello. I also noticed the same warning about the deprecated command in a recent PR.

Warning: The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

FWIW, based on the details in that blog post I think the changes in this PR should be alright. Could you please take a look before the commands are disabled at the end of the month? Thanks.

**Review by CMDJojo — APPROVED** [review]
I think the changes look good. It would be great to merge this before June 1st, because if it doesn't get merged until then, the action will fail (according to GitHub docs)

**Pull request merged**
